astropy/io/fits/tests/test_hdulist.py

```diff
@@ -1040,12 +1040,14 @@ def test_pop_extname(self):
             assert len(hdul) == 5
             assert hdu_popped is hdu1
 
+    # Skip due to ttps://github.com/astropy/astropy/issues/8916
```

Comment:
Missing the h in front of the link.

Reply:
Good catch. Arghh... Maybe I should go to bed before attempting more coding.